### DISCUSSION TITLE
Use relative path in index.html for production using vite

### DISCUSSION THREAD

**Discussion by kirschkern** [quasarframework/quasar#13928] [General - Components / Directives / etc] [answered]
When building my quasar/vite project for production using `quasar build`, all file references in index.html starts with "/". Now all referenced files cause a 404 when I put my app in a subfolder like `"https://domain.com/subfolder/"` and the app doesn't start.

I've tried several values for `build.publicPath` in `quasar.config.js` but all relative URLs will have the "/" prefixed. Removing publicPath or set it to null or undefined, doesn't change anything. It only works when using absolute URLs (like starting with https://). 
But I want to use relative URLs as the app can be at different locations.

All my quasar/webpack based projects work without the leading "/".

Any hint, what I can do to use relative URLs?

Other relevant setting: vueRouterMode: 'hash'
I'm using the latest quasar version 2.7.5 with @quasar/app-vite - 1.0.5.

**Comment by alexrififi** [comment, marked as answer]
Try [Public Base Path ](https://vitejs.dev/guide/build.html#public-base-path)

**Reply by kirschkern** [reply]
Great, that works.
My quasar.config.js now includes the following

    build: {
        // ... other stuff

        extendViteConf (viteConf) {
            viteConf.base = "";
        },

       // ... more stuff
    }
